src/z3c/schema/baseurl/field.py

```diff
@@ -39,7 +39,7 @@ class BaseURL(zope.schema.URI):
 
     def _validate(self, value):
         if isValidBaseURL(value) and value.endswith('/') and \
-            not value.endswith(':/'):
+                not value.endswith(':/'):
```

Comment:
```suggestion
                and not value.endswith(':/'):
```

Since this is being changed anyway...Shouldn't the `and` go on the same line as the `not`? Breaking in the middle of the operation seems weird and introduces a strange mental pause, at least for me.

Reply:
Yes it should go there. I just ran `autopep8` and did not have the steam to look at each individual change.
Thank you for taking a closer look.